app/controllers/gift_requests_controller.rb

```diff
@@ -0,0 +1,41 @@
+class GiftRequestsController < ApplicationController
+  def new
+    @gift_request = GiftRequest.new
+    render "gift_request", locals: {list_id: params[:id]}
```

Comment:
have you tested this? my understanding of nested routing says that `lists/1/gift_request/new` should put the '1' in `params[:list_id]`.